File: CS/CustomViewInfoRegistrator.cs
using DevExpress.XtraNavBar;
using DevExpress.XtraNavBar.ViewInfo;

namespace DXApplication99
{

    public class CustomViewInfoRegistrator : SkinNavigationPaneViewInfoRegistrator {
        public override string ViewName { get { return "MyView"; } }

        public override NavBarViewInfo CreateViewInfo(NavBarControl navBar) {
            var info = new SkinNavigationPaneViewInfoEx(navBar);
            return info;
        }
    }
}

File: VB/CustomViewInfoRegistrator.vb
Imports DevExpress.XtraNavBar
Imports DevExpress.XtraNavBar.ViewInfo

Namespace DXApplication99

    Public Class CustomViewInfoRegistrator
        Inherits SkinNavigationPaneViewInfoRegistrator

        Public Overrides ReadOnly Property ViewName() As String
            Get
                Return "MyView"
            End Get
        End Property

        Public Overrides Function CreateViewInfo(ByVal navBar As NavBarControl) As NavBarViewInfo
            Dim info = New SkinNavigationPaneViewInfoEx(navBar)
            Return info
        End Function
    End Class
End Namespace